src/style.css

```diff
@@ -1,3 +1,86 @@
 body {
   margin: 0;
+  padding: 0;
+  display: block;
+}
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo_image {
+  width: 40px;
+  height: 40px;
+}
+
+.navigation {
+  align-items: center;
+  display: flex;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0;
+  padding-left: 0;
+}
+
+.nav-item {
+  text-align: center;
+  display: flex;
+}
```

Comment:
The class name `.nav-item` should be consistent with other class names like `.nav_list` and `.nav_link`. Use `.nav__item` to follow the BEM (Block Element Modifier) naming convention.